config/locales/en.yml

```diff
@@ -1349,4 +1349,6 @@ en:
     select_all: Select all
     cancel: Cancel
     apply: Apply
-    unselect_all: Unselect all
\ No newline at end of file
+    unselect_all: Unselect all
+  copy_original_uri: Copy original URI
```

Comment:
add the in the ` components:` section of this file. and to call them in the code you will do `t('components.copy_original_uri')`